feature/stamps/src/main/java/io/github/droidkaigi/confsched2023/stamps/section/StampList.kt

```diff
@@ -21,39 +33,75 @@ private const val DoubleItemSpanCount = 2 / 2
 @Composable
 fun StampList(
     stamps: ImmutableList<Stamp>,
-    onStampsClick: () -> Unit,
+    @androidx.annotation.RawRes
+    stampLottieRawId: Int?,
+    onStampsClick: (Stamp) -> Unit,
+    onReachAnimationEnd: () -> Unit,
     modifier: Modifier = Modifier,
 ) {
-    LazyVerticalGrid(
-        columns = Fixed(StampListColumns),
-        modifier = modifier,
-        contentPadding = PaddingValues(
-            horizontal = 16.dp,
-            vertical = 20.dp,
+    val bgColor: Color by animateColorAsState(
+        if (stampLottieRawId != null) {
+            Color(0xFF37383D)
+        } else {
+            Color.Transparent
+        },
+        animationSpec = tween(
+            delayMillis = 2,
```

Comment:
In Figma, there is an instruction to specify 1.5 ms.
However, delayMillis can only specify an Int value, so it is set to 2.
I am not sure if this is correct, so please point out if I am wrong.🙇‍♂️

Reply:
Thank you for your report and we'll talk about this issue with design's side.
But I think your solution makes sense 😉

Reply:
We discussed the animation you implemented with our designer.
It's a great implementation and there are no design concerns 👍